packages/cortex-react/src/components/Accordion.tsx

```diff
@@ -0,0 +1,151 @@
+import clsx from 'clsx';
+import React, {
+  createContext,
+  useCallback,
+  useContext,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react';
+import { LiaAngleRightSolid } from 'react-icons/lia';
+
+interface AccordionProps {
+  children?: React.ReactNode;
```

Comment:
Adicionar JSDoc para children 
/** child element **/
Sem o JSDoc (por conta do autodocs) ele vai omitir o children, com o JSDoc ele força aparecer essa prop